.github/workflows/release-docker-canary.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish Packages (canary)
+name: Publish Docker images (canary)
```

Comment:
This only publishes docker now. NPM releases are published everytime we merge to develop

Reply:
We will want to validate this works as soon as it merges and that it didn't have some hidden dependency on the npm workflow running.  It shouldn't though because all the other releases are purely 100% docker